org.osgi.test.common/src/main/java/org/osgi/test/common/annotation/InjectBundleInstaller.java

```diff
@@ -59,4 +59,4 @@
 })
 @Retention(RUNTIME)
 @Documented
```

Comment:
This is missing a `@Target(FIELD, PARAMETER)` declaration (it appears that this error was pre-existing).

Reply:
It is just hidden by GitHub.  Hit the EXPAND button on line 57-59 ;-)